### ISSUE TITLE
Broken on Python 2.6

### ISSUE THREAD

**Issue by eevee** [jiffyclub/snakeviz#45]
```
  File "snakeviz/stats.py", line 51
    return {keyfmt(*k): v for k, v in d.items()}
                            ^
SyntaxError: invalid syntax
```

Dict comprehensions are new in 2.7.

**Comment by jiffyclub** [comment]
That's right, thanks for the report. I'll fix when I get a chance...

**Comment by jiffyclub** [comment]
Thanks again for reporting this, it's fixed in the 0.3 release.
